src/Microsoft.ML.Transforms/doc.xml

```diff
@@ -39,14 +41,344 @@
+       </summary>
+      <remarks>
+        Scaling inputs to unit norms is a common operation for text classification or clustering.
+        For more information see: <a href="http://www.cs.stanford.edu/~acoates/papers/coatesleeng_aistats_2011.pdf"></a>
+      </remarks>
+      <seealso cref=" Microsoft.ML.Transforms.LpNormalizer"></seealso>
+      <example>
+        <code language="csharp">
+          pipeline.Add(new GlobalContrastNormalizer(&quot;FeatureCol&quot;){ SubMean= false });
+        </code>
+      </example>
+    </member>
+    
+  <member name="Ungroup">
+      <summary>
+        Un-groups vector columns into sequences of rows, inverse of Group transform.
+       </summary>
+      <remarks>
+        <para>This can be thought of as an inverse of the CombinerByContiguousGroupId. 
+        For all specified vector columns ("pivot" columns), performs the "ungroup" (or "unroll") operation as outlined below.
+        </para>
+        <para>If the only pivot column is called P, and has size K, then for every row of the input we will produce 
+         K rows, that are identical in all columns except P. The column P will become a scalar column, and this 
+         column will hold all the original values of input's P, one value per row, in order. The order of columns 
+         will remain the same.
+        </para>
+        <para>Variable-length pivot columns are supported (including zero, which will eliminate the row from the result).</para>
+        <para>Multiple pivot columns are also supported:</para>
+        <list type="bullet">
+          <item><description>A number of output rows is controlled by the 'mode' parameter. 
+            <list type="bullet">
+              <item>outer<description> it is equal to the maximum length of pivot columns</description></item>
+              <item>inner<description> it is equal to the minimum length of pivot columns</description></item>
+              <item>first<description> it is equal to the length of the first pivot column</description></item>
+            </list>
+            </description>
+          </item>
+          <item><description>
+              If a particular pivot column has size that is different than the number of output rows, the extra slots will
+              be ignored, and the missing slots will be 'padded' with default values.
+            </description></item>
+        </list>
+        <para>All metadata is preserved for the retained columns. For 'unrolled' columns, all known metadata
+        except slot names is preserved.
+        </para>
+      </remarks>
+    </member>
+    <example name="Ungroup">
+      <example>
+        <code language="csharp">
+          pipeline.Add(new Segregator(){ Column = new[]{&quot;Column1&quot; }, Mode = UngroupTransformUngroupMode.First} );
+        </code>
+      </example>
+    </example>
+
+    <member name="KeyToText">
+      <summary>
```

Comment:
>summary [](start = 7, length = 7)

I suggest
```
Transforms keys in a key column into the corresponding values
```
The fact that it is using the metadata looks like implementation detail to me